util/regtool.py

```diff
@@ -192,6 +196,18 @@ def main():
         log.error(str(err))
         exit(1)
 
+    # If this block has countermeasures, we grep for RTL annotations in all
+    # .sv implementation files and check whether they match up with what is
+    # defined inside the Hjson.
+    annotation_list = []
+    if obj.countermeasures:
+        sv_files = parentdir.joinpath('rtl').glob('*.sv')
+        for sv_file in sv_files:
+            with open(sv_file) as f:
+                annotation_list += CounterMeasure.from_rtl_file(f.name,
+                                                                f.read())
```

Comment:
With my suggestions above, this loop turns into `CounterMeasure.search_rtl_files(sv_files)`.

Reply:
Yes I can do that.
I mainly did it that way since we're handling all the file IO at this level otherwise, and just pass file contents down into the classes.